storage/store_test.go

```diff
@@ -1348,13 +1357,18 @@ func (fq *fakeRangeQueue) MaybeRemove(rng *Range) {
 // TestMaybeRemove tests that MaybeRemove is called when a range is removed.
 func TestMaybeRemove(t *testing.T) {
 	defer leaktest.AfterTest(t)
-	store, _, stopper := createTestStore(t)
+	store, _, stopper := createTestStoreWithoutStart(t)
 	defer stopper.Stop()
```

Comment:
do you think this line should follow `store.Start()`? do we still need to call `stopper.Stop()` if `store.Start()` fails?